src/themes/elife/styles.css

```diff
@@ -316,6 +316,34 @@
       text-transform: capitalize;
     }
   }
+
+  :--Figure {
+    margin: 1em 0;
+
+    img {
+      outline: 1px solid #e0e0e0;
+      padding: 0.75rem;
+    }
+
+    :--Heading {
+      font-size: var(--TEXT-SIZE-SMALL-MEDIUM-REM);
+      line-height: 1.84615;
+      padding-bottom: 0;
+    }
+
+    :--Paragraph {
+      font-size: var(--TEXT-SIZE-SMALL-MEDIUM-REM);
+      line-height: 1.84615;
+    }
+  }
+
+  :--label {
+    display: block;
+    font-size: var(--TEXT-SIZE-MEDIUM-REM);
+    font-weight: bold;
+    line-height: 1.71429;
+    padding: 0.75rem 0;
```

Comment:
This appears as it should to me. The only issue I am unsure on is whether we care that the `padding-top` is on the `h4` rather than the `figcaption`.

Reply:
The padding-top is on the global H3 and H4 elements so seems a bit excess to remove it from the h4 to the fiig-caption itself. Do you agree?